src/target/riscv/riscv-013_reg.c

```diff
@@ -358,30 +737,21 @@ int riscv013_reg_examine_all(struct target *target)
  */
 int riscv013_reg_save(struct target *target, enum gdb_regno regid)
```

Comment:
If you like, this could be named `riscv013_reg_save_gpr` to make it very clear that the function operates with GPRs only.

```suggestion
int riscv013_reg_save_gpr(struct target *target, enum gdb_regno regid)
```

The comment on top of the function could also be updated to reflect that:

```
/**
 * This function is used to save the value of a general purpose register (GPR)
 * into the cache. The register is marked as dirty, and writeback is delayed
 * for as long as possible.
 */
```

Reply:
Addressed